### ISSUE TITLE
Remove authentication on log-out button click.

### ISSUE THREAD

**Issue by alodahl** [Techtonica/keyboard-shortcuts-practice#80]
Remove authentication on log-out button click.

**Comment by snarvekark** [comment]
I would like to work on this issue

**Comment by vegetabill** [comment]
@snarvekark sorry for the confusion this issue is blocked until Priority 5 so I'm going to unassign you at the moment in case you want to work on a different issue. But you can have priority later to work on it when it's unblocked.

**Comment by snarvekark** [comment]
No worries, I will work on another issue

**Comment by vegetabill** [comment]
This will be done by Auth0 or the fake auth so it shouldn't need any specific work.
